Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1210,7 +1210,10 @@ def lhegensim(fragment,howMuch):
 steps['RECOUP15_trackingOnlyLowPU']=merge([step3_trackingLowPU, step3Up2015Defaults_trackingOnly]) # todo: remove UP from label
 steps['RECOUP15_HIPM']=merge([step3_HIPM,step3Up2015Defaults]) # todo: remove UP from label
 
-# for Run2 PPb workflows
+# for Run1 PPb data workflow
+steps['RECO_PPbData']=merge([{'-s':'RAW2DIGI,L1Reco,RECO,ALCA:TkAlMinBias+TkAlMuonIsolatedPA+TkAlUpsilonMuMuPA+TkAlZMuMuPA,SKIM:ZMM+ZEE+MinBias,EI,DQM','--scenario':'pp','--conditions':'auto:run2_data','--era':'Run1_pA','--datatier':'AOD,DQMIO','--eventcontent':'AOD,DQM'}, dataReco])
```

Comment:
auto:run2_data better be auto:run1_data at least for semantic purpose (the two are identical since a while)